extensions/amp-smartlinks/amp-smartlinks.md

```diff
@@ -0,0 +1,102 @@
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-smartlinks" src="https://cdn.ampproject.org/v0/amp-smartlinks-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>nodisplay</td>
+  </tr>
+</table>
+
+## Overview
+
+ At [Narrativ](https://narrativ.com/), we transform static commerce links into dynamic, multimerchant nodes. With a library of millions of products matched to expert reviews from top commerce publishers, we lift publisher revenue through real-time bidding and data solutions.
+
+This AMP extension is our Linkmate service in AMP. See the full documentation for Linkmate [here](http://docs.narrativ.com/en/stable/linkmate.html).
+
+## Getting started
+
+Your account must be a member of our Linkmate program to use this feature. For more information about this program, feel free to contact your account manager or [hello@narrativ.com](mailto:hello@narrativ.com).
+
+NOTE: If you plan to use `amp-smartlinks` alongside other affiliate partners you will need to specify the meta tag shown below. The tag will specify the order in which the affiliate tags fire.
+
+In your AMP page you will have to add the following snippets:
+
+```html
+<!doctype html>
+<html ⚡>
+<head>
+  ...
+  <meta name="amp-link-rewriter-priorities" content="amp-smartlinks amp-other-affiliate">
+  <script async custom-element="amp-smartlinks" src="https://cdn.ampproject.org/v0/amp-smartlinks-0.1.js"></script>
+  ...
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+    ...
+    <amp-smartlinks
+        layout="nodisplay"
+        nrtv-account-name="supercoolpublisher"
+        linkmate="true">
+    </amp-smartlinks>
+    ...
+</body>
+</html>
+```
+
+## Attributes
+
+<table>
+  <tr>
+    <td class="col-fourty"><strong>nrtv-account-name</strong></td>
+    <td><strong>Required</strong></td>
+    <td>Your Narrativ account name given to you by your account manager. Need to know your Narrativ account name? Log into <a href="https://dashboard.narrativ.com/#/login">dashboard.narrativ.com</a> and go to <a href="https://dashboard.narrativ.com/#/publisher/account/setup">setup</a> to see your account name in the snippet, or reach out to your account manager for support as needed.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong>linkmate</strong></td>
+    <td><strong>Required</strong></td>
+    <td>Field to flag if you want Linkmate to run on an article. Expected values are <code>true</code> or <code>false</code>.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong>exclusive-links</strong></td>
+    <td><strong>Optional</strong></td>
+    <td>Field to flag if you want exclusive links on an article. Expected values are <code>true</code> or <code>false</code>.</td>
```

Comment:
The protoascii encodes that the only allowed value is an empty string.  I think that's the right approach (see my comment in the protoascii file), but this document doesn't match up.